src/react/components/styleform/SizeControl.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Message } from 'oskari-ui';
+import { Form, InputNumber } from 'antd';
+
+const sizeFormatter = (number) => Math.abs(number); 
+
+
+export const SizeControl = (props) => {
+    const locKey =
+        props.format === 'point' ? 'VisualizationForm.point.size.label'
+        : props.format === 'line' ? 'VisualizationForm.line.width.label'
+        : 'VisualizationForm.area.linewidth.label'
```

Comment:
Make SizeControl take the label as child component or prop so you don't have to tie the label to the style logic:
```
<SizeControl ...props>
   <Message bundleKey={ props.locSettings.localeKey } messageKey={ locKey }
</SizeControl>
```